rabbitmq/test_rabbitmq.py

```diff
@@ -183,6 +183,8 @@ def test_connections(self):
         self.run_check(CONFIG, force_reload=True)
         self.assertMetric('rabbitmq.connections', tags=['rabbitmq_vhost:/', "tag1:1", "tag2"], value=2, count=1)
         self.assertMetric('rabbitmq.connections', count=1)
+        self.assertMetric('rabbitmq.connections.state', tags=['rabbitmq_conn_state:running'], value=2, count=1)
+        self.assertMetric('rabbitmq.connections.state', count=1)
```

Comment:
could you assert the same test in the next two `run_check` ? You can set `count=0` to assert that a metric was not sent.